src/plugins/intel_gpu/include/intel_gpu/graph/network.hpp

```diff
@@ -194,6 +206,12 @@ struct network {
         return *_memory_pool;
     }
 
+    /// Assigns memory state locations
+    void assign_memory_states(memory_states_map&& memory_states);
```

Comment:
I stick to already existing terminology in [CPU](https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/src/infer_request.cpp#L125) and [GNA](https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_gna/gna_plugin.cpp#L1528) plugin. Do you think GPU plugin should have different from other plugins naming?

Reply:
There is no need to align with the local struct name with cpu and gna plugin. I am not sure how "memory" term is importantly used in those plugin, but I think "memory" is too general in GPU plugin. We'd better to set the naming to show the purpose. Since this memory is only for the variable, I think variable is suitable.